nix-update.el

```diff
@@ -76,12 +77,13 @@
                          (when (looking-at "^\\(\\s-+\\)")
                            (setq leader (match-string 1)))
                          (goto-char (line-end-position))
-                         (insert ?\n leader field " = " value ";")))))
+                         (insert ?\n leader field " = \"" value "\";")))))
             (let ((data
                    (pcase type
                      (`"fetchFromGitHub"
                       (let ((owner (get-field "owner"))
                             (repo (get-field "repo"))
+                            (rev (or (get-field "rev") "refs/heads/master"))
```

Comment:
The problem with using `"refs/heads/master"` as a hardcoded string is that it
will break where people are using other default branch names such as `main`.

In that case, I had success with just specifying an empty string, which
`nix-prefetch-git` could handle well, so maybe just use that.

I know this is not something you changed, I just wanted to leave that note.

Reply:
Thanks for the tip @nagy! Didn't give this too much priority since it doesn't look like it'll get merged, but I just tested your suggestion and it works!